Resources/Private/Fusion/Root.fusion

```diff
@@ -1,8 +1,50 @@
-// this is only included in dev context
+prototype(PackageFactory.AtomicFusion.PropTypes:Int) {
+  @class = 'PackageFactory\\AtomicFusion\\PropTypes\\FusionObjects\\IntImplementation'
+}
+
+prototype(PackageFactory.AtomicFusion.PropTypes:Float) {
+  @class = 'PackageFactory\\AtomicFusion\\PropTypes\\FusionObjects\\FloatImplementation'
+}
+
+prototype(PackageFactory.AtomicFusion.PropTypes:Bool) {
+  @class = 'PackageFactory\\AtomicFusion\\PropTypes\\FusionObjects\\BoolImplementation'
+}
+
+prototype(PackageFactory.AtomicFusion.PropTypes:String) {
+  @class = 'PackageFactory\\AtomicFusion\\PropTypes\\FusionObjects\\StringImplementation'
+  regularExpression = null
+}
+
+prototype(PackageFactory.AtomicFusion.PropTypes:ArrayOf) {
+  @class = 'PackageFactory\\AtomicFusion\\PropTypes\\FusionObjects\\ArrayOfmplementation'
+}
+
+prototype(PackageFactory.AtomicFusion.PropTypes:AnyOf) {
+  @class = 'PackageFactory\\AtomicFusion\\PropTypes\\FusionObjects\\AnyOfmplementation'
+}
+
+prototype(PackageFactory.AtomicFusion.PropTypes:OneOf) {
+  @class = 'PackageFactory\\AtomicFusion\\PropTypes\\FusionObjects\\OneOfmplementation'
+}
+
+prototype(PackageFactory.AtomicFusion.PropTypes:DataStructure) {
+  @class = 'PackageFactory\\AtomicFusion\\PropTypes\\FusionObjects\\DataStructureImplementation'
+}
+
+prototype(PackageFactory.AtomicFusion.PropTypes:InstanceOf) {
+  @class = 'PackageFactory\\AtomicFusion\\PropTypes\\FusionObjects\\InstanceOfImplementation'
+  type = null
+}
+
+prototype(PackageFactory.AtomicFusion.PropTypes:ForPrototype) {
+  @class = 'PackageFactory\\AtomicFusion\\PropTypes\\FusionObjects\\ForPrototypeImplementation'
+  prototypeName = null
+}
+
 prototype(Neos.Fusion:Component) {
-    @exceptionHandler = 'PackageFactory\\AtomicFusion\\PropTypes\\Error\\ExceptionHandler\\PropTypeExceptionHandler'
+  @exceptionHandler = 'PackageFactory\\AtomicFusion\\PropTypes\\Error\\ExceptionHandler\\PropTypeExceptionHandler'
```

Comment:
i had the same dilemma : these exception handlers will now be used also in production ...

Reply:
Yes but they will let everything but PropTypeException pass so they should do no harm. And the propType exceptions are not thrown in production unless it is activated.

Reply:
i see 👍